doc/source/data/doc_code/transforming_datasets.py

```diff
@@ -384,16 +366,6 @@ def convert_to_numpy(df: pd.DataFrame) -> Dict[str, np.ndarray]:
     }
 
 ds = ds.map_batches(convert_to_numpy)
-# -> Dataset(
```

Comment:
Can we restore these string representations?

Reply:
@ericl - sure, added back.

Reply:
btw found a separate issue for string representation, fixing in https://github.com/ray-project/ray/pull/32411 .